pulp_ansible/app/tasks/collections.py

```diff
@@ -280,6 +282,20 @@ def create_collection_from_importer(importer_result):
     return collection_version
 
 
+def _get_backend_storage_url(artifact_file):
```

Comment:
I feel pulpcore would be the right place for this, but we can have it on pulp_ansible by now

Reply:
I agree. If we can also open it against pulpcore and keep it here so pulp_ansible doesn't require a pulpcore upgrade that would be ideal. @awcrosby can you also contribute this to pulpcore? @daviddavis I think can guide where/how to contribute it if he agrees.

Reply:
Sounds good, I added a pulpcore issue to track this separately: https://pulp.plan.io/issues/8521